AdminNotificationManagerPlugin.inc.php

```diff
@@ -15,25 +15,11 @@
 
 class AdminNotificationManagerPlugin extends GenericPlugin {
 
-    
-    	/**
-	 * @var $currentOjsVersion object
-	 * 
-	 * This string holds the current version object returned by the VersionDAO
-	 * object. It's built in $this->register() and is used throughout the plugin
-	 * to support backwards compatibility with older versions of OJS.
-	 */
-	public $currentAppVersion = null;
-
         /**
 	 * @copydoc Plugin::register()
 	 */
 	public function register($category, $path, $mainContextId = null) {
-		// Setting version information for backwards compatibility in other areas of the plugin
-		$versionDao = DAORegistry::getDAO('VersionDAO');
-		$this->currentAppVersion = $versionDao->getCurrentVersion();
-
-                $success = parent::register($category, $path, $mainContextId);
+    $success = parent::register($category, $path, $mainContextId);
```

Comment:
This spacing still doesn't look quite right (four spaces instead of one tab).  You can add a commit here to fix it, or you can merge it and then fix it in #9.